### ISSUE TITLE
Return missed UID

### ISSUE THREAD

**Issue by raufbulbul** [Codefans-fan/zklib#7]
if uid is 350, zkMachine.py returning 35, if uid is 445, zkMachine.py returning 44. last digits is missing ?

**Comment by kurenai-ryu** [comment]
mind to try my library? https://github.com/kurenai-ryu/pyzk

**Comment by raufbulbul** [comment]
Yes. Because only your library has Realtime events for attendance systems :)

Android için Outlook<https://aka.ms/ghei36> uygulamasini edinin

________________________________

**Comment by hamzaafridi** [comment]
@kurenai-ryu is your implementation TCP based?

**Comment by kurenai-ryu** [comment]
yes, it tries to connect via TCP and fallbacks to UDP if it can't connect via TCP.
